### DISCUSSION TITLE
Strange error in backup log

### DISCUSSION THREAD

**Discussion by riaanp78** [vmware-tanzu/velero#5858] [Community support Q&A]
Does anyone perhaps know what this error is about?

```
time="2023-02-14T10:41:10Z" level=info msg="1 errors encountered backup up item" backup=velero/mysql-test2 logSource="pkg/backup/backup.go:421" name=wordpress-mysql-66ffb96555-6lzdp
time="2023-02-14T10:41:10Z" level=error msg="Error backing up item" backup=velero/mysql-test2 error="pod volume backup failed: error creating uploader: failed to connect repository: error running command=restic snapshots --repo=s3:http://192.168.1.100:9000/velero-config/restic/mysql-test --password-file=/tmp/credentials/velero/velero-repo-credentials-repository-password --cache-dir=/scratch/.cache/restic --latest=1 --insecure-tls=true, stdout=, stderr=Fatal: unable to open config file: Stat: The specified key does not exist.\nIs there a repository at the following location?\ns3:http://192.168.1.100:9000/velero-config/restic/mysql-test\n: exit status 1" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/podvolume/backupper.go:223" error.function="[github.com/vmware-tanzu/velero/pkg/podvolume.(*backupper).BackupPodVolumes](http://github.com/vmware-tanzu/velero/pkg/podvolume.(*backupper).BackupPodVolumes)" logSource="pkg/backup/backup.go:425" name=wordpress-mysql-66ffb96555-6lzdp
```

Trying to understand why that happens. The MinIO S3 server is external and it does work (others are using it),

**Comment by walexharm23** [comment]
Hi,
Churchillson Markus here,
I'm experiencing the same problem; when I go to my [womens leather credit card holder](https://leatherjild.com/product/womens-handmade-leather-credit-card-holder/) dashboard, I receive the message (can't access admin dashboard).
I would appreciate it if you could let me know if the answer is correct.
I appreciate your help.

**Comment by cuttingwarmer567** [comment]
Hello,
It's me, CWarmer,
Whenever I try to access my[ Mini Militia Ranks List](https://minimilitiahackapk.net/mini-militia-ranks-list-badges/) dashboard, I receive the message (can't access admin dashboard).
Please let me know if the answer is correct.
You have been a great help to me.

**Comment by alvisinwalex678** [comment]
Hi there,
My name is Alvisin Walex
I receive the message (Cannot access admin dashboard) when trying to access my [best casual basketball shoes](https://myindoorbasketball.com/best-basketball-shoes-for-casual-wear/) dashboard.
I would appreciate it if you could let me know if the answer is correct.
Thank you for taking the time to work with me.

**Comment by jacksonville234** [comment]
Hello there,
My name is Jackson
I receive the message (can't access admin dashboard) whenever I try to access my site https://gofinanc.com/ dashboard.
Could you please let me know if the answer is correct.
You have been a great help to me.

**Comment by jerryharmer567** [comment]
Hi,
jerry here,
When I access my [los angeles security company](https://ausguards.com/) dashboard, I receive the message (can't access admin dashboard).
It would be greatly appreciated if you could let me know if the answer is correct.
Your assistance is greatly appreciated.

**Comment by elmenkarm879** [comment]
Hi there,
Hi, I'm Elmen,
When I try to access my dashboard at https://carparkingmultiplayermodapk.net/, I receive the message (can't access admin dashboard).
I would appreciate it if you could let me know if the answer is correct.
You have been a great help to me.

**Comment by neonmalwer456** [comment]
Hi there!
I'm Neon. I'm having trouble accessing my dashboard at [virtual sales assistant](https://sharkstack.co/virtual-sales-assistant) - every time I try, I get an error message saying "can't access admin dashboard". I would really appreciate it if you could let me know if you have any ideas on how to resolve this issue. You've already been a great help to me!

**Comment by balveryuter23** [comment]
Hi, my name is Balver. When I try to access the admin dashboard for my [instander mod apk download](https://apkgmp.com/social/instander-mod-apk/) through the provided link.I receive an error message saying that I can't access it. I would greatly appreciate it if you could let me know if you have any solutions to this issue. Thank you for your assistance.

**Comment by jalvergaze45** [comment]
Hi, 
 When I attempt to use the provided link to log into the admin dashboard for my client's https://slidesharedown.com/. I get a notice that informs me that I cannot access it. If you could let me know if you have any suggestions for how to resolve this problem, I'd be very grateful. I appreciate you helping me.

**Comment by yulvenpetex** [comment]
Hi,
When I try to access the admin dashboard for my client's https://freefiregamedownload.com/ using the provided link. I receive a notification telling me I can't access it. I would greatly appreciate it if you could let me know if you have any ideas on how to tackle this issue. I value your assistance.

**Comment by datvertalwer23** [comment]
Hi,
When I use the supplied link to enter the admin dashboard for my client's https://slidesharedownloading.com/. I get a notification that I'm unable to access it. If you could let me know if you have any suggestions for how to approach this problem, I would be very grateful. I appreciate you helping me.

**Comment by denvertulex** [comment]
Hi,
I try to use the supplied link to access the admin dashboard for my [graphic designer in exeter](https://jhtackettmarketing.com/graphic-designer-in-exeter/) download.I get a notice that informs me that I cannot access it. If you could let me know if you have any suggestions for how to resolve this problem, I'd be very grateful. I appreciate you helping me.

**Comment by gatveralber** [comment]
In the backup log, there is an error encountered while backing up an item related to a pod volume backup. The error message indicates that there was a failure in creating the uploader and connecting to the specified repository location. The command used to run the backup encountered an issue where it was unable to open the configuration file, resulting in a "The specified key does not exist" error. The error includes the location of the repository, which in this case is "s3:http://192.168.1.100:9000/velero-configresticmysql-test". The exit status of the command was 1, indicating a failure. This error is logged in the file "backupper.go" at line 223 in the Velero package.A [certified translation of a marriage certificate](https://lingoservice.com/marriage-certificate-translation/) refers to a translated version of the certificate that has been authenticated and verified by a professional translator or translation service. This type of translation is typically required when presenting a marriage certificate in a different language or for legal purposes in a foreign country. It ensures that the translated document accurately reflects the original content and can be recognized as valid and reliable.
